### ISSUE TITLE
The Dice scores are low using my model.

### ISSUE THREAD

**Issue by luqily** [MIC-DKFZ/TractSeg#64]
Hi, I am sorry to bother you again.
I have used HCP_105 dataset to training my model. The following is the image of metrics during training.
![image](https://user-images.githubusercontent.com/43931240/67617447-dbf85f00-f815-11e9-9d3d-4fc0f5ca9b16.png)

The hyperparamters are as follows.
![image](https://user-images.githubusercontent.com/43931240/67617811-9ccc0d00-f819-11e9-8363-da2c5b801aad.png)
![image](https://user-images.githubusercontent.com/43931240/67617532-938d7100-f816-11e9-931f-d59cd30573be.png)
![image](https://user-images.githubusercontent.com/43931240/67617533-94260780-f816-11e9-87a4-6a337906577b.png)
![image](https://user-images.githubusercontent.com/43931240/67617576-fb43bc00-f816-11e9-9e72-71540f21c5bb.png)

Then I have used `ExpRunner --test=False --train=False --seg --lw --cofig=My_custom_experiment` to segment the fibers. But the Dice scores are low.
![image](https://user-images.githubusercontent.com/43931240/67617609-4c53b000-f817-11e9-977f-d2884f67a3f6.png)

The code for calculating Dice is as follows.
![image](https://user-images.githubusercontent.com/43931240/67617625-80c76c00-f817-11e9-9503-5f497a234d21.png)
![image](https://user-images.githubusercontent.com/43931240/67617763-1d3e3e00-f819-11e9-8499-c0de377ae005.png)

And the segmented images have a large deviation.
![image](https://user-images.githubusercontent.com/43931240/67617671-1c58dc80-f818-11e9-9da7-36ca6f0f9cfa.png)
The top line is my result. And the bottom line is ground truth. From left to right is  `AF_left`, `AFT_right`, `ATR_left` and `ATR_right`.

I guess there's something wrong with my steps, but I don't know where the mistake is.

**Comment by wasserth** [comment]
It might be the case that you are comparing the wrong bundles. It seems that in the top and bottom line you are not showing the same bundles. 

You can also test your trainined model the following way for one subject. This way you will get one segmentation file for each bundle. Might be less confusing.
```
TractSeg  -i my_peaks.nii.gz --exp_name My_custom_experiment
```

**Comment by luqily** [comment]
> It might be the case that you are comparing the wrong bundles. It seems that in the top and bottom line you are not showing the same bundles.
> 
> You can also test your trainined model the following way for one subject. This way you will get one segmentation file for each bundle. Might be less confusing.
> 
> ```
> TractSeg  -i my_peaks.nii.gz --exp_name My_custom_experiment
> ```
I have saved the images of each beam directly. 
![image](https://user-images.githubusercontent.com/43931240/67672109-29113800-f9b2-11e9-88ca-d92de1a585e2.png)
And the code to get the bundle masks is as follows.
![image](https://user-images.githubusercontent.com/43931240/67672260-87d6b180-f9b2-11e9-94a0-968535cd4f07.png)
I think they correspond. 

What's more,` EXPRIMENT_TYPE=tract_segmentation`, right?

**Comment by luqily** [comment]
> It might be the case that you are comparing the wrong bundles. It seems that in the top and bottom line you are not showing the same bundles.
> 
> You can also test your trainined model the following way for one subject. This way you will get one segmentation file for each bundle. Might be less confusing.
> 
> ```
> TractSeg  -i my_peaks.nii.gz --exp_name My_custom_experiment
> ```

I think it's not in alignment for every slice.
 
![image](https://user-images.githubusercontent.com/43931240/67768694-07857e80-fa8e-11e9-95e7-3e17ca555a7e.png)

But I don't know how to align them.
